package.json

```diff
@@ -36,14 +37,15 @@
     "karma-sourcemap-loader": "^0.3.2",
     "karma-webpack": "^1.3.1",
     "mocha": "^2.0.1",
-    "react": "0.13.0-rc1",
+    "react": "^0.13.0",
```

Comment:
need to change this to `0.13.x` so that people don't accidentally get `0.14` before we're ready (like right now, today, if we did `^0.13.0` the router would be completely broken on initial installs.